webpack.common.config.js

```diff
@@ -128,7 +128,8 @@ var resolve = {
         'diffTool': staticPath('js/diffTool.js'),
         'mathrender': staticPath('js/mathrender.js'),
         'citations': staticPath('js/citations.js'),
-        'tests': staticPath('js/tests')
+        'tests': staticPath('js/tests'),
+        'projectOrganizer': staticPath('js/projectorganizer.js')
```

Comment:
This doesn't necessarily have to change now, but I think we should add an alias `osf` which maps to `staticPath('js')` to use as a namespace for internal modules. We would then be able to do:

```javascript
var mathrender = require('osf/mathrender')
var projectOrganizer = require('osf/projectorganizer');
//...
```

and we wouldn't create a new alias every time a new module is added.